dependabot/update_dependencies_in_pipeline.py

```diff
@@ -178,37 +179,49 @@ def wait_for_current_level_build(level):
             sys.exit(1)
 
     module_release_failure = False
+    chat_message = ""
     pr_checks_failed_modules = list(
         filter(lambda s: s[MODULE_CONCLUSION] == MODULE_CONCLUSION_PR_CHECK_FAILURE, current_level_modules))
     if len(pr_checks_failed_modules) != 0:
         module_release_failure = True
         print('Following modules dependency PRs have failed checks...')
+        chat_message += 'Following modules dependency PRs have failed checks...' + "\n"
```

Comment:
Optimise so that the print statements can be removed and the final chat message can be printed in one go. Also add a final statement to the chat message to trigger the workflow of Update dependencies and give the url